scene/main/node.cpp

```diff
@@ -1364,6 +1364,30 @@ bool Node::has_node(const NodePath &p_path) const {
 	return get_node_or_null(p_path) != nullptr;
 }
 
+Node *Node::get_unique_node(const StringName &p_name) const {
+	if (!p_name) {
+		return nullptr;
+	}
+
+	if (data.owned_unique_nodes.size()) {
+		// Has unique nodes in ownership
+		Node **unique = data.owned_unique_nodes.getptr(p_name);
```

Comment:
`data.owned_unique_nodes.getptr(p_name)` here actually references the const function making it return a `Node *const *` instead of `Node **`, this needs to be fixed by making it being `Node *const *unique = data.owned_unique_nodes.getptr(p_name);`

Reply:
It's ok, how gentle! I figured just at the same time as this reply. It all compiles now.